SONIC-DEV/sonic-buildimage
# [201911] Update the changes from upstream

**- What I did**
Update 201911_cel_wb to meet the current upstream head.
**- How I did it**
Pull change from Azure/201911
**- How to verify it**
Build image and apply Questone2BD device and test on UUT. 
Test the SDK and see if the shell command is running OK.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
